rust/xaynet-analytics/src/database/common.rs

```diff
@@ -1,20 +1,28 @@
-use anyhow::Error;
+use anyhow::{anyhow, Error, Result};
```

Comment:
just as a side note for future refactoring: we should replace the generic `anyhow!` errors in the crate with specific error types, for example derived from the `IsarError` type. there are some nice patterns to do this without much boilerplate with the help of the `thiserror` and `displaydoc` crates.

Reply:
Yes, thanks, agree. We discussed it a bit with Andrea already. For now if it's OK i would leave this as it was easier to get things to work first. Then we'll optimise and clean it up